### DISCUSSION TITLE
Feeding base camp *nitpicks*

### DISCUSSION THREAD

**Discussion by LordNIl11** [CleverRaven/Cataclysm-DDA#46355] [General]
Problem
Currently you can ditch prepared mutant toxic food into larders  (cracklins from mutaded fat and jerky is good example) or abuse barrels of alcohol as food source-with them yielding  hundreds of thousands of calories greatly reducing pressure of feeding lot of npcs...

Solution
players should not be allowed to abuse npcs with feeding them suboptimal food player would avoid himself or keeping them alive on booze...

Alternatives
Keep feeding crap to npcs and making them subsist on alcholol.Its easier for players including me;)

**Comment by actual-nh** [comment]
I note that No_NPC_Food is a [default mod](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/mods/default.json); feeding them such is actually doing more than needed by default...

**Reply by anothersimulacrum** [reply]
This is base camp feeding, which is required with or without the mod (food is payment).

**Reply by actual-nh** [reply]
Oops. This tells you exactly how far I've managed to get while playing...

**Comment by Jamuro-g** [comment]
How about instead of blocking these excesses we just treat faction npcs like the player in this regard.
Track the average fun of the calories stored/used for base jobs and make that a morale modifier affecting project durations.
